django/river/topicleaner/service.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from time import sleep
+
+from django.utils import timezone
+
+from river import models
```

Comment:
Not really related to this file bue DecrementingCounter doesn't work, I'll change the adapter so that we use 2 counters as before